hplsql/src/test/queries/local/dbms_output.sql

```diff
@@ -3,4 +3,7 @@ DECLARE
 BEGIN
   DBMS_OUTPUT.PUT_LINE('Hello, world!');
   DBMS_OUTPUT.PUT_LINE(str);
+  DBMS_OUTPUT.PUT_LINE(trim('a''a'));
+  DBMS_OUTPUT.PUT_LINE(trim('''a'));
+  DBMS_OUTPUT.PUT_LINE(trim('a'''));
```

Comment:
Removed trim function from test cases. 
It will throw error as single quotes are not properly closed.
Added the test cases for the specified scenarios.